vasil-testnet/cardano-node.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 exec cardano-node run \
-  --config config/config.json \
-  --topology config/topology.json \
+  --config cardano-node/config.json \
+  --topology cardano-node/topology.json \
   --database-path db \
-  --socket-path ipc/node.socket
```

Comment:
why this changed?

Reply:
This directory and these scripts were first created by hand. I did this and other changes to make it more consistent with a directory created by the now existing `hydra-cluster` executable. So you can use `hydra-cluster --vasil-dev --state-directory vasil-testnet` as starting point for the smoke test and no new files should be created.